src/components/dataDisplays/DataDisplay.jsx

```diff
@@ -257,10 +258,22 @@ export default function DataDisplay({
         style={paperStyles}
       >
         <Table
-          style={{ minWidth: 10 }}
+          style={{ minWidth: 10, ...tableStyles }}
           size={tableSize}
           aria-label={title}
         >
+          <colgroup>
+            {columns.map(c => (
+              <col
+                key={c.name}
+                style={
+                  get(c, 'options.width')
+                    ? { width: c.options.width }
+                    : {}
+                }
+              />
+            ))}
+          </colgroup>
```

Comment:
This will create `colgroup` and `col` tags for every DataDisplay component. Because they are empty they do not affect the table's appearance.

**NOTE:** this logic will need to be updated if a table with an "expand" row column wants to set custom column widths.

Reply:
It is allowing me to set custom width values for columns in the table. Without it, the columns would each take approximately the same amount of horizontal space. It looks really strange when one column has clipped text and there is a large section of white space before the next column's content.

I do think that this logic will probably undergo some changes if or as we apply noWrap to other tables.